### PR TITLE
ChatGPT - Case Study

### PR DESCRIPTION
**Pull request by Melisa-Zorraindo**

### CODE REVIEW

File: chatGPT-case-study.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+The model also presents limitations in terms of server capacity. It is at times unreachable due to the amount of people trying to use the service at the same time. This is being addressed by offering a subscription plan.
+
+Furthermore, it cannot make internet searches to find new sources of information, being limited to the data it was used during its training. Although this was massive, the training finished in 2021 which makes it of little use to get information on topics that are constantly updated. In the same line it cannot provide data on current affairs.
+
+## Comparison
+
+ChatGPT is not the first chat bot in the market. Most company's customer service departments offer their users the possibility to solve their problems in real-time by chatting with a bot. Although they use NLP and AI to understand what the customer says, they don't perform exceptionally well and more often than not the interaction leads to frustration for the human being. Their knowledge, moreover, is limited to the task they're meant to carry out. ChatGPT, however, is not constrained to a specific activity, as it was trained using a large amount of data in different topics and styles.
```
Comment (Fermain) [inline chatGPT-case-study.md:84]:
```suggestion
ChatGPT is not the first chat bot in the market. Most companies' customer service departments offer their users the possibility to solve their problems in real-time by chatting with a bot. Although they use NLP and AI to understand what the customer says, they don't perform exceptionally well and more often than not the interaction leads to frustration for the human being. Their knowledge, moreover, is limited to the task they're designed to carry out. For example, [a car insurance chatbot can't do trigonometry, etc] ChatGPT, however, is not constrained to a specific activity, as it was trained using a large amount of data in different topics and styles.
```

File: chatGPT-case-study.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Another significant improvement was made with the introduction of the Moderation API. The API help developers protect their applications against possible misuse by giving them free access to GPT-based classifiers that detect undesired content. Bots like Tay and Blenderbot 3 would not have failed (at least so quickly) if they had been provided with a similar tool. ChatGPT is able to refuse to engage in offensive behaviour more efficently than other similar bots. https://openai.com/blog/chatgpt/ shows a comparison between ChatGPT and InstructGPT where the different responses to the same malicious question reveal a clear advance.
```
Comment (Fermain) [inline chatGPT-case-study.md:86]:
```suggestion
Another significant improvement was made with the introduction of the Moderation API. The API help developers protect their applications against possible misuse by giving them free access to GPT-based classifiers that detect undesired content. Bots like Tay and Blenderbot 3 would not have failed (at least so quickly) if they had been provided with a similar tool. ChatGPT is able to refuse to engage in offensive behaviour more efficiently than other similar bots. [This post from OpenAI](https://openai.com/blog/chatgpt/) shows a comparison between ChatGPT and InstructGPT where the different responses to the same malicious question reveal a clear advance.
```

File: chatGPT-case-study.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+## Summary
+
+Many have expressed their concern about ChatGPT and other AI tools built on large language models taking their jobs. As technology improves, these models will be capable of better understand irony, sarcarsm, jokes, human emotions, and the nuances introduced by cultural differences making them extremely powerful and suitable for sectors like psychiatry and medicine.
+
+As far as software development is concern, ChatGPT can write code faster than a human, but not more reliably. It requires human supervision to ensure it's working as intended. Even if you ask the chat bot to automate testing, these tests may also contain bugs. Replacing a human proves not to be so easy after all. Instead, this type of technology is most useful to boost productivity - to assist human beings in their daily tasks. It is obviously more efficient to ask a question and obtain a response in seconds than to post it on stackoverflow and wait for other developers to read our code and provide an answer. It is also easier to read one response than sift through dozens of pages from a Google search. This is why replacing an entire staff with bots is unlikely to happen in the near future.
```
Comment (Fermain) [inline chatGPT-case-study.md:94]:
```suggestion
As far as software development is concerned, ChatGPT can write code faster than a human, but not more reliably. It requires human supervision to ensure it's working as intended. Even if you ask the chat bot to automate testing, these tests may also contain bugs. Replacing a human proves not to be so easy after all. Instead, this type of technology is most useful to boost productivity - to assist human beings in their daily tasks. It is obviously more efficient to ask a question and obtain a response in seconds than to post it on stackoverflow and wait for other developers to read our code and provide an answer. It is also easier to read one response than sift through dozens of pages from a Google search. This is why replacing an entire staff with bots is unlikely to happen in the near future.
```

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a really well researched and constructed case study about a subject that I find captivating. I have made some edits and suggestions, most of these are grammatical, a few tone and the rest are prompts.

Putting these aside, you have met the brief exactly as expected and delivered a high quality report.

File: chatGPT-case-study.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+Due to the human trainers rewarding longer answers for their appearance of being more comprehensive, the bot tends to be excessively verbose. This doesn't precisely add quality to the responses since the same information is repeated in different paragraphs with a slight change in phrasing. I personally consider this to be its biggest giveawayas it makes it relatively easy for a human to realize when a text has been written by an AI tool and therefore not reliable for producing important texts for university or work.
+
+The model also presents limitations in terms of server capacity. It is at times unreachable due to the amount of people trying to use the service at the same time. This is being addressed by offering a subscription plan.
+
+Furthermore, it cannot make internet searches to find new sources of information, being limited to the data it was used during its training. Although this was massive, the training finished in 2021 which makes it of little use to get information on topics that are constantly updated. In the same line it cannot provide data on current affairs.
```
[the remark this answers is not in the document]
Comment (Melisa-Zorraindo) [reply chatGPT-case-study.md]: added

File: chatGPT-case-study.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+Additionally, it tends to be unpredictable and unstable. This means that users may avoid content moderation by repeatedly asking the same question in different ways. It is rumoured that one user was even provided with detailed instructions on how to make a molotov cocktail.
+
+Due to the human trainers rewarding longer answers for their appearance of being more comprehensive, the bot tends to be excessively verbose. This doesn't precisely add quality to the responses since the same information is repeated in different paragraphs with a slight change in phrasing. I personally consider this to be its biggest giveawayas it makes it relatively easy for a human to realize when a text has been written by an AI tool and therefore not reliable for producing important texts for university or work.
+
+The model also presents limitations in terms of server capacity. It is at times unreachable due to the amount of people trying to use the service at the same time. This is being addressed by offering a subscription plan.
```
[the remark this answers is not in the document]
Comment (Melisa-Zorraindo) [reply chatGPT-case-study.md]: Added

File: chatGPT-case-study.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+Focusing on software development, its ability to increase developers productivity is definitely its greatest strength. If the tool is used appropriately it can help us be more efficient by decreasing the time we spent planning or reading how to do something. It is especially useful for students or developers in the early stage of their careers. Finding the answer that solves an issue requires knowledge of technical terms that students often forget or don't yet have - having the opportunity to explain the problem in simple terms is therefore a great advantage.
+
+The above applies as well when learning to read errors. Errors may look like gibberish at first and you have to learn to interpret them. Being provided with an explanation that beginner developers can understand more easily helps bridging that gap.
```
[the remark this answers is not in the document]
Comment (Melisa-Zorraindo) [reply chatGPT-case-study.md]: Added

File: chatGPT-case-study.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+## Strengths
+
+ChatGPT can be universally used. It isn't limited to English speakers for it supports many different languages. Users' age doesn't appear to be a limitation either given its ability to explain complex concepts in simple terms. Additionally, it can assist people in their personal life by outlying meal plans or help organise their days, as well as it can aid in a variety of industries like journalism, education, business development, and software development. This makes it a very versatile tool.
+
+Focusing on software development, its ability to increase developers productivity is definitely its greatest strength. If the tool is used appropriately it can help us be more efficient by decreasing the time we spent planning or reading how to do something. It is especially useful for students or developers in the early stage of their careers. Finding the answer that solves an issue requires knowledge of technical terms that students often forget or don't yet have - having the opportunity to explain the problem in simple terms is therefore a great advantage.
```
[the remark this answers is not in the document]
Comment (Melisa-Zorraindo) [reply chatGPT-case-study.md]: Added

**Comment by Melisa-Zorraindo** [comment]
This PR is not yet ready to merge. I'll add a link to a practical example of chatGPT used in the context of frontend development and update this PR soon.

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a good quality case study about a compelling subject. There is only one issue, you are deleting the tensorflow case study! This will remove someone else's work from the main branch. Please remerge main and un-delete this file.

**Pull request merged**